### DISCUSSION TITLE
Re order difficulties of a mapset when DT / HR / DA changes the order they should be in.

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#16776] [Ideas] [answered]
some very specific types of maps, (mainly long stream maps with DT and high CS maps with HR) can have massive differences in final values which leave the lists feeling messy and nonsensical, now as it stands this isnt the biggest issue since most mapsets only have like 5 difficulties, but when maps become sortable without being inside their own sets this would be a pretty large consideration

**Comment by frenzibyte** [comment]
Is this proposing for beatmaps to be ordered by their original star difficulty, not the post-mods one? not 100% sure about that... is it something osu!stable does?

**Reply by bdach** [reply]
that's the current lazer behaviour, no? and opposite what is wanted as far as i understand from the opening post.

**Reply by frenzibyte** [reply]
My mind just blew up.

**Comment by frenzibyte** [comment, marked as answer]
Turns out I was reading this the opposite way. Tracking under https://github.com/ppy/osu/issues/12628, as they're both referring to the `CarouselBeatmap` ordering by the static `StarRating` property instead of the difficulty cache.

**Reply by mk56-spn** [reply]
Yeah that was it. Sorry if I didn't explain it well enough.
